### PR TITLE
Add multiple gateways to Actility LoraWan provider

### PR DESCRIPTION
**Pull request by BobClaerhout**
Signed-off-by: Bob Claerhout <claerhout.bob@gmail.com>

### CODE REVIEW

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java [hunk 1]
```diff
@@ -83,4 +84,20 @@ public void extractRssiFromLoraMessage() {
...
+     * Verifies that the extracted gateways are correct.
+     */
+    @Test
+    public void extractGatewaysFromLoraMessage() {
+        final JsonObject loraMessage = LoraTestUtil.loadTestFile("actility.uplink");
+        final Map<String, Object> map = provider.extractNormalizedData(loraMessage);
+        Assert.assertTrue(map.containsKey(LoraConstants.GATEWAYS));
+
+        JsonArray expectedArrays =
+                new JsonArray("[{\"gateway_id\":\"18035559\",\"rss\":48.0,\"snr\":3.0},{\"gateway_id\":\"18035560\"," +
```
Comment (sophokles73) [inline adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java]: can you please create the JSON objects in the array using `JsonObject`?

Comment (BobClaerhout) [reply adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java]: I could have expected this :)

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java [hunk 2]
```diff
@@ -83,4 +83,21 @@ public void extractRssiFromLoraMessage() {
...
+    /**
+     * Verifies that the extracted gateways are correct.
+     */
+    @Test
+    public void extractGatewaysFromLoraMessage() {
+        final JsonObject loraMessage = LoraTestUtil.loadTestFile("actility.uplink");
+        final Map<String, Object> map = provider.extractNormalizedData(loraMessage);
+        Assert.assertTrue(map.containsKey(LoraConstants.GATEWAYS));
+
+        JsonArray expectedArray = new JsonArray();
```
Comment (sophokles73) [inline adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java]: needs to be final

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java [hunk 3]
```diff
@@ -83,4 +83,21 @@ public void extractRssiFromLoraMessage() {
...
+        expectedArray.add(new JsonObject().put("gateway_id", "18035559").put("rss", 48.0).put("snr", 3.0));
+        expectedArray.add(new JsonObject().put("gateway_id", "18035560").put("rss", 49.0).put("snr", 4.0));
+
+        JsonArray gateways = new JsonArray(map.get(LoraConstants.GATEWAYS).toString());
```
Comment (sophokles73) [inline adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java]: final

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java [hunk 4]
```diff
@@ -83,4 +83,21 @@ public void extractRssiFromLoraMessage() {
...
+        Assert.assertEquals(expectedArray,gateways);
```
Comment (sophokles73) [inline adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java]: missing whitespace after `,`

**Comment by sophokles73** [comment]
you should install the checkstyle plugin in Eclipse and let it check your code before pushing it ....

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java (hunk 2 above)
Comment (BobClaerhout) [reply adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java]: done

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java (hunk 3 above)
Comment (BobClaerhout) [reply adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java]: done

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java (hunk 4 above)
Comment (BobClaerhout) [reply adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/ActilityProviderTest.java]: done

**Comment by BobClaerhout** [comment]
Installed it now, I was under the assumption the checkstyle was performed when running `mvn test` as well. I was executing this before each push

**Pull request merged**